R/r.R

```diff
@@ -108,19 +109,32 @@ rename_files <- function(old, new) {
   old <- path_ext_remove(old)
   new <- path_ext_remove(new)
 
-  # Move .R file
+  # R/ ------------------------------------------------------------------------
   r_old_path <- proj_path("R", old, ext = "R")
   r_new_path <- proj_path("R", new, ext = "R")
   if (file_exists(r_old_path)) {
     ui_done("Moving {ui_path(r_old_path)} to {ui_path(r_new_path)}")
     file_move(r_old_path, r_new_path)
   }
 
+  # src/ ------------------------------------------------------------------------
+  if (dir_exists(proj_path("src"))) {
+    src_old <- dir_ls(proj_path("src"), glob = glue("*/{old}.*"))
```

Comment:
```suggestion
    src_old <- dir_ls(proj_path("src"), glob = glue("*/src/{old}.*"))
```
Feels a bit more consistent with the assumptions I think are in the `gsub()`, i.e. we're only dealing with files in top-level of `src/`.

Reply:
Ooooh I thought the glob was relative to the first argument.